indexer/src/main/java/au/org/aodn/esindexer/service/GeoNetworkServiceImpl.java

```diff
@@ -94,7 +88,7 @@ public String findGroupById(String uuid) throws IOException {
             Map<String, Object> params = new HashMap<>();
             params.put("id", group);
 
-            HttpEntity<String> requestEntity = getRequestEntity(MediaType.APPLICATION_JSON, null);
+            HttpEntity<String> requestEntity = geoNetworkRequestEntityCreator.getRequestEntity(MediaType.APPLICATION_JSON, null);
```

Comment:
Seems no need to re-create object every time as they are the same?

Reply:
this will be deleted as mentioned here: https://github.com/aodn/es-indexer/pull/63#discussion_r1572109165